operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md

```diff
@@ -77,6 +77,18 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.
+
+For example: 
+> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
+>
+>We are here to [support](https://link-to-support-section) you.
+
+as opposed to
+> Deploying the Mattermost Mobile App will take time and iteration.
+
+
+
 **Voice**
 
 Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it.
```

Comment:
Is this accurate for Customer Education content?

Reply:
I'm not 100% sure here, but as of now I'd say yes.